pom.xml

```diff
@@ -41,7 +41,7 @@
 
     <properties>
         <!-- Compile libs -->
-        <fastjson.version>1.2.58</fastjson.version>
+        <fastjson.version>1.2.60</fastjson.version>
```

Comment:
Since it's not related to the changes about the dashboard, could you please submit another new PR to make this change (in another branch)?

Reply:
ok

Reply:
> Since it's not related to the changes about the dashboard, could you please submit another new PR to make this change (in another branch)?

refer to #1045

Reply:
👍